bitnami/mariadb-galera/templates/svc.yaml

```
@@ -8,18 +8,10 @@ kind: Service
 metadata:
   name: {{ template "common.names.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
-    {{- end }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
   {{- if or .Values.service.annotations .Values.commonAnnotations }}
-  annotations:
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.service.annotations }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.annotations "context" $) | nindent 4 }}
-    {{- end }}
+  {{- $annotations := merge .Values.headless.annotations .Values.commonAnnotations }}
```

I guess you refer to the old value here instead of `headless`

```suggestion
  {{- $annotations := merge .Values.service.annotations .Values.commonAnnotations }}
```

---

done at https://github.com/bitnami/charts/pull/18325/commits/71740fc1671a5e5e09921b617da0a885975576e4